README.md

```diff
@@ -2,24 +2,20 @@
 [![Build
 Status](https://travis-ci.org/helm/monocular.svg?branch=master)](https://travis-ci.org/helm/monocular)
 
-Monocular is web-based UI for managing Kubernetes applications packaged as Helm
-Charts. It allows you to search and discover available charts from multiple
-repositories, and install them in your cluster with one click.
+Monocular is a web-based UI and API that enables the search and discovery of
```

Comment:
Maybe a higher level description would make more sense, i.e `Monocular is a web-based **application** that enables the search and discovery of...`